### DISCUSSION TITLE
[IntelliSense] Capability to mark certain function arguments as candidates for tailwindcss autocomplete

### DISCUSSION THREAD

**Discussion by raphaelvigee** [tailwindlabs/tailwindcss#11022] [Ideas]
When declaring a utils function, it would be nice to have autocomplete:
```
function classNames(type: MessageType) {
    switch (type) {
        case 'danger':
            return 'bg-red-100 border-red-400';
        case 'info':
            return 'bg-blue-100 border-blue-400';
    }
}

```

**Comment by wongjn** [comment]
You could try configuring the [`tailwindCSS.experimental.classRegex` setting](https://github.com/tailwindlabs/tailwindcss/issues/7553#issuecomment-735915659).

**Reply by raphaelvigee** [reply]
Oh did not realise this is what its meant for, works, thx!
